adding Flow .json file, validating on DS Langflow (#10)
crtr0/rag-sf-app: README.md

Movies++ is a movie recommendation application that makes use of [GenAI](https://en.wikipedia.org/wiki/Generative_artificial_intelligence) to recommend movies based on natural language input. It is built on [DataStax Astra](https://astra.datastax.com/) and was demoed at [CascadiaJS](https://www.youtube.com/live/HfsNGyDQtJ4?si=XzDN5lzEcmIXncJ7&t=30203) and [DataStax's RAG++ AI Hack Night](https://www.datastax.com/events/rag-plus-plus-ai-hack-night-june-2024).

## Working with Langflow

> To use RAG with Langflow, you'll need to run Langflow. You can either do this as a [hosted cloud solution](https://langflow.datastax.com) on DataStax, or follow the [Langflow documentation](https://docs.langflow.org/) to get started running it locally.

Reviewer: ```suggestion
To use RAG with Langflow, you'll need to run Langflow. You can either do this as a [hosted cloud solution](https://langflow.datastax.com) on DataStax, or follow the [Langflow documentation](https://docs.langflow.org/getting-started-installation) to get started running it locally.
```

## Getting Started

To get started with this project and run it locally, follow the steps below:

1. Clone the repository
2. Change directory (`cd`) into the cloned repository
3. Install dependencies with `pnpm install`
4. In your Astra database:
   - Create a vector-enabled collection named `movies` in the namespace `default_keyspace`

>      - For "Embedding generation method", select (and configure if not already) "Open AI Embedding Generation"

Reviewer: ```suggestion
     - For "Embedding generation method", select (and [configure if not already](https://docs.datastax.com/en/astra-db-serverless/integrations/embedding-providers/openai.html)) "OpenAI Embedding Generation"
```

       - Select the `text-embedding-3-small` model
       - Leave Dimensions unchanged (`1536`)
       - Change Similarity Metric to `Dot Product`
5. Import `Movies RAG.json` into your Langflow instance, and do the following:
   - On the two Astra DB components, set:
     - Collection (should be named `movies`) 
     - Astra DB Application Token (generate a token if you do not have one)
     - Database (or API Endpoint if you are running Langflow locally)

>    - On the two Open AI Embeddings components, and the OpenAI chat component:

Reviewer: ```suggestion
   - On the two OpenAI Embeddings components, and the OpenAI chat component:
```

# Navigate to our scripts
cd ./scripts

> # Create and activate a venv (optional)

Reviewer: ```suggestion
# Create and activate a venv (optional, but recommended)
```
